azext_hanaonazure/modules_sdk/models/_models.py

```diff
@@ -489,9 +489,11 @@ class SapMonitor(Resource):
     :type hana_db_username: str
     :param hana_db_password: Database password of the HANA instance.
     :type hana_db_password: str
-    :param hana_db_password_key_vault_url: URL to the KeyVault secret that contains the HANA credentials.
```

Comment:
you might need to rebase?

Reply:
I already did, not sure why it is removing that

Reply:
The description was changed in PR https://github.com/Azure/azure-rest-api-specs/commit/79ecf24dcb870e4b293a8fef7a597a2cd1340d12#diff-5d31a3020218d4446080e6fa4795f10e, so I guess my Python SDK update caught that change